setup.py

```diff
@@ -27,5 +23,14 @@ def package_files(directory):
         'torch>=1.9.0, <2.0.0',
         'torchvision>=0.10.0, <1.0.0'
     ],
+}
+
+setup(
+    name='alibi-testing',
+    version='0.0.11',
```

Comment:
Bump version for increased clarity? Otherwise LGTM!

Reply:
@jklaise you ok with version `0.1.0` since this is a pretty notable change?

Reply:
Sure, why not

Reply:
Ok will merge now then. Cheers!